client/ayon_core/hosts/hiero/api/otio/hiero_import.py

```diff
@@ -397,28 +396,21 @@ def create_trackitem(playhead, track, otio_clip, clip):
         source_in = source_range.end_time_inclusive().value
 
         timeline_in = playhead + source_out
-        timeline_out = (
-            timeline_in +
-            source_range.duration.value
-        ) - 1
+        timeline_out = (timeline_in + source_range.duration.value) - 1
```

Comment:
Do we still care about keeping the brackets?
```suggestion
        timeline_out = timeline_in + source_range.duration.value - 1
```

Reply:
(There are a few more lines of code where this is the case in this PR)